ecs-cli/modules/cli/compose/project/project_test.go

```diff
@@ -17,331 +17,50 @@ import (
-	}
-	if len(web.Entrypoint) != 1 || entryPoint != web.Entrypoint[0] {
-		t.Errorf("Expected entryPoint to be [%s] but got [%s]", entryPoint, web.Entrypoint)
-	}
-
-	sort.Strings(env)
-	webEnv := []string{}
-	for _, val := range web.Environment {
-		webEnv = append(webEnv, val)
-	}
-	sort.Strings(webEnv)
-	if !reflect.DeepEqual(env, webEnv) {
-		t.Errorf("Expected Environment to be [%v] but got [%v]", env, webEnv)
-	}
-	if !reflect.DeepEqual(extraHosts, web.ExtraHosts) {
-		t.Errorf("Expected extraHosts to be [%v] but got [%v]", extraHosts, web.ExtraHosts)
-	}
-	if hostname != web.Hostname {
-		t.Errorf("Expected Hostname to be [%s] but got [%s]", hostname, web.Hostname)
-	}
-	if len(labels) != len(web.Labels) ||
-		labels["label1"] != web.Labels["label1"] || labels["com.foo.label2"] != web.Labels["com.foo.label2"] {
-		t.Errorf("Expected labels to be [%v] but got [%v]", labels, web.Labels)
-	}
-	if len(web.Links) != 1 || !reflect.DeepEqual(links[0], web.Links[0]) {
-		t.Errorf("Expected links to be [%v] but got [%v]", links, web.Links)
-	}
-	if logDriver != web.Logging.Driver {
-		t.Errorf("Expected logDriver to be [%s] but got [%s]", logDriver, web.Logging.Driver)
-	}
-	if !reflect.DeepEqual(logOpts, web.Logging.Options) {
-		t.Errorf("Expected logOpts to be [%v] but got [%v]", logOpts, web.Logging.Options)
-	}
-	if memLimit != int64(web.MemLimit) {
-		t.Errorf("Expected memLimit to be [%d] but got [%d]", memLimit, web.MemLimit)
-	}
-	if !reflect.DeepEqual(ports, web.Ports) {
-		t.Errorf("Expected ports to be [%v] but got [%v]", ports, web.Ports)
-	}
-	if privileged != web.Privileged {
-		t.Errorf("Expected privileged to be [%t] but got [%t]", privileged, web.Privileged)
-	}
-	if readonly != web.ReadOnly {
-		t.Errorf("Expected readonly to be [%t] but got [%t]", readonly, web.ReadOnly)
-	}
-	if !reflect.DeepEqual(securityOpts, web.SecurityOpt) {
-		t.Errorf("Expected securityOpts to be [%v] but got [%v]", securityOpts, web.SecurityOpt)
-	}
-	if user != web.User {
-		t.Errorf("Expected user to be [%s] but got [%s]", user, web.User)
-	}
-	if len(volumes.Volumes) != len(web.Volumes.Volumes) {
-		t.Errorf("Expected len of volumes to be [%d] but got [%d]", len(volumes.Volumes), len(web.Volumes.Volumes))
-	}
-	if !reflect.DeepEqual(*volumes.Volumes[0], *web.Volumes.Volumes[0]) {
-		t.Errorf("Expected volumes to be [%v] but got [%v]", volumes.Volumes[0], web.Volumes.Volumes[0])
-	}
-	if workingDir != web.WorkingDir {
-		t.Errorf("Expected workingDir to be [%s] but got [%s]", user, web.WorkingDir)
-	}
-
-}
-
-func TestParseComposeForVersion2Files(t *testing.T) {
-	wordpressImage := "wordpress"
-	mysqlImage := "mysql"
-	ports := []string{"80:80"}
-	memoryReservation := yaml.MemStringorInt(500000000)
-	memory := yaml.MemStringorInt(536870912)   // 512 * 1024 * 1024
-	shmSize := yaml.MemStringorInt(1073741824) // 1 gb = 1024 * 1024 * 1024 bytes
-	tmpfs := yaml.Stringorslice{"/run:size=1gb", "/tmp:size=65536k"}
-
+func TestParseCompose(t *testing.T) {
```

Comment:
since we're providing a V2 compose string, should we call this test 'TestParseComposeWithV2'?

Reply:
I could -- the idea for this test was to assert things that would be true regardless of file version (hence no assertions on output container configs, since those are tested in the parsev3 and parsev1v2 files.